### PR TITLE
[provisioning] enable benchtop SIVAL cert endorsement with Nitrokeys

### PR DESCRIPTION
**Pull request by timothytrippel**
The DICE CA key may be stored on a Nitrokey, which can be used in a benchtop provisioning flow to endorse DICE certificates. This enables such flows by simply setting an envar (`PKCS11_MODULE_PATH`) and plugging in a Nitrokey to the host machine.

### CODE REVIEW

File: sw/device/silicon_creator/manuf/keys/sival/ca_config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "dice": {
+    "certificate": "sv00-earlgrey-a1-ca-dice-0",
+    "key_id": "0x5398A4F090F2A95C3D52FC98DEB9F2F9AF042F6E",
+    "key_type": "Token",
+    "key": "sv00-earlgrey-a1-ca-dice-0"
+  },
+  "ext": {
```
Comment (cfrantz) [inline sw/device/silicon_creator/manuf/keys/sival/ca_config.json:8]: TODO: make `ext` optional for skus that don't have extra certs.

File: sw/device/silicon_creator/manuf/keys/sival/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The real (private) keys used for the SIVAL SKU are stored on offline HSMs. The
+matching public keys and certificates are checked into the repository.
+
+To use the private keys to endorse the certificates in benchtop provisioning
+flow, one must set the `PKCS11_MODULE_PATH` envar to point to the PKCS#11 shared
+library for the hardware token they are using. For example, if the private keys
+are stored on a Nitrokey, you would set the envar as follows:
+```sh
+export PKCS11_MODULE_PATH=/opt/nitrokey/lib/libsc-hsm-pkcs11.so
```
Comment (cfrantz) [inline sw/device/silicon_creator/manuf/keys/sival/README.md:16]: Note: you can use `opensc-pkcs11.so` here.  The debian package `opensc-pkcs11` installs it as `/usr/lib/x86_64-linux-gnu/opensc-pkcs11.so`.

**Review by cfrantz — APPROVED** [review]

**Review by moidx — APPROVED** [review]
nit: You may have to add the keys/sival data dependencies to orchestrator if you are planning to use it without bazel.

**Review by milesdai — APPROVED** [review]

File: sw/device/silicon_creator/manuf/keys/sival/README.md (hunk 2 above)
Comment (timothytrippel) [reply sw/device/silicon_creator/manuf/keys/sival/README.md:36]: Updated README. Thanks.

**Comment by timothytrippel** [comment]
> nit: You may have to add the keys/sival data dependencies to orchestrator if you are planning to use it without bazel.

Ah right, fixed. Thanks.

**Pull request merged**